static/styles/portico/portico-signin.scss

```diff
@@ -501,6 +501,12 @@ html {
     .inline-block {
         text-align: left;
     }
+
+    .contact-admin p.invite-hint {
+        font-size: 1.3rem;
+        margin-top: 0.8rem;
```

Comment:
How did you pick this sizing?

Reply:
I selected this size so that it will easily be noticed.
I don't find any other place where we use such type of messages to get the font idea.
Do you think it need to be changed?

For org description, we use font-size of 17px and font-weight of 400, we can try that styling too, that can also work(although I don't have a very solid reason why should we use the same)
![Screenshot from 2020-04-07 11-41-12](https://user-images.githubusercontent.com/32801674/78635841-b68f7700-78c4-11ea-94ef-5538ed16cadd.png)